jujugui/static/gui/src/app/components/shared/status/unit-list/unit-list.js

```diff
@@ -94,7 +98,7 @@ class StatusUnitList extends React.Component {
           content: this._formatPorts(unit.portRanges)
         }, {
           columnSize: 2,
-          content: unit.workloadStatus.message
+          content: (unit.workloadStatus || {}).message
```

Comment:
I'm a little confused (an admittedly common state of affairs). Will all these places do the right thing with `undefined`? Or am I misunderstanding what e.g. `{}.message` will resolve to if e.g. `unit.workloadStatus` is falsey?